app/src/sharedTest/java/org/oppia/android/app/customview/LessonThumbnailImageViewTest.kt

```diff
@@ -138,12 +146,43 @@ class LessonThumbnailImageViewTest {
       .check(matches(withDrawable(R.drawable.topic_fractions_01)))
   }
 
+  @Module
+  class TestModule {
+    @Provides
+    @SplashScreenWelcomeMsg
+    fun provideSplashScreenWelcomeMsgParam(): PlatformParameterValue<Boolean> {
+      return PlatformParameterValue.createDefaultParameter(SPLASH_SCREEN_WELCOME_MSG_DEFAULT_VALUE)
+    }
+
+    @Provides
+    @SyncUpWorkerTimePeriodHours
+    fun provideSyncUpWorkerTimePeriod(): PlatformParameterValue<Int> {
+      return PlatformParameterValue.createDefaultParameter(
+        SYNC_UP_WORKER_TIME_PERIOD_IN_HOURS_DEFAULT_VALUE
+      )
+    }
+
+    @Provides
+    @EnableLanguageSelectionUi
+    fun provideEnableLanguageSelectionUi(): PlatformParameterValue<Boolean> {
+      return PlatformParameterValue.createDefaultParameter(false)
+    }
+
+    @Provides
+    @EnableEditAccountsOptionsUi
+    fun provideEnableEditAccountsOptionsUi(): PlatformParameterValue<Boolean> {
+      return PlatformParameterValue.createDefaultParameter(
+        false
+      )
+    }
+  }
+
   // TODO(#59): Figure out a way to reuse modules instead of needing to re-declare them.
   @Singleton
   @Component(
     modules = [
       RobolectricModule::class,
-      PlatformParameterModule::class, PlatformParameterSingletonModule::class,
+      TestModule::class, PlatformParameterSingletonModule::class,
```

Comment:
Why isn't PlatformParameterModule sufficient (for here & other tests)? If it includes the new parameter, I wouldn't expect these tests to actually need to redefine the values unless they specifically need to change them (i.e. to enable the feature).

Reply:
Yes, I too expected the same behaviour. But for some reason these files aren't able to consume the Platform Parameter Values from `PlatformParameterModule`....But only when I define the functions generating parameter values locally inside the Test, i.e, in a  `TestModule`, the tests finally succeed.